### PR TITLE
Hide promoted tweets

### LINKED ISSUE #3: Hide promoted tweets and ads
I wanted to add this at launch, but I never encountered a promoted tweet or ad when developing, even with my ad-blocker disabled. If anyone encounters promoted tweets or ads, please dump the HTML into a Gist and comment the link here. Or even better, do a pull request implementing this.

### PR DESCRIPTION
**Pull request by elyager**
I added the code to remove promoted tweets, I hope is correct at least is working for me 😄

The second open source project that I do a small contribution, so let me know any corrections.

✌🏼

---

Fixes #3

### CODE REVIEW

File: source/content.js [hunk 1]
```diff
@@ -6,6 +6,10 @@ import autoLoadNewTweets from './features/auto-load-new-tweets';
 // Add global for easier debugging
 window.$ = $;
 
+function cleanPromotedTweets() {
+	$('#stream-items-id .promoted-tweet').remove();
```
Comment (sindresorhus) [inline source/content.js]: You don't need `#stream-items-id`. The class is specific enough.

File: source/content.js [hunk 2]
```diff
@@ -6,6 +6,10 @@ import autoLoadNewTweets from './features/auto-load-new-tweets';
 // Add global for easier debugging
 window.$ = $;
 
+function cleanPromotedTweets() {
```
Comment (sindresorhus) [inline source/content.js]: `cleanPromotedTweets` => `hidePromotedTweets`

**Review by sindresorhus — CHANGES_REQUESTED** [review]

**Comment by sindresorhus** [comment]
Looking good! 👌😄

Can you add something like `- Hides promoted tweets` to https://github.com/sindresorhus/refined-twitter#highlights ?

File: source/content.js (hunk 1 above)
Comment (sindresorhus) [reply source/content.js]: And you should probably remove the parent item, as that's the actual container, just to be safe. So:

```js
$('.promoted-tweet').parent().remove();
```

**Comment by sindresorhus** [comment]
You also need to fix the merge conflict ;)

**Comment by elyager** [comment]
I finally got the time to get this quick fix done.

**Review by sindresorhus — APPROVED** [review]

**Comment by sindresorhus** [comment]
Looks great. Thanks for contributing! :)

**Pull request merged**